.circleci/config.yml

```
@@ -0,0 +1,42 @@
+orbs:
+  architect: giantswarm/architect@2.1.0
+
+version: 2.1
+workflows:
+  build_and_e2eTest:
```

Just a nit, but since there's no e2e test then maybe just:

```suggestion
  build:
```